### DISCUSSION TITLE
Livewire with selec2 options from database?

### DISCUSSION THREAD

**Discussion by rhaider-dev** [livewire/livewire#1210] [4. General]
I have a select2 field in livewire component. I want to search product name from database in the select2 and bind the option value with a livewire model. Is it possible? I have tried but getting null result every time.

**Comment by ATholin** [comment]
Hey @rhaider-dev, please provide code to reproduce your issue

**Reply by rhaider-dev** [reply]
I have solved the issue using event listener. Thanks
